map/bookmark_manager.cpp

```diff
@@ -2326,6 +2340,15 @@ kml::MarkGroupId BookmarkManager::CreateBookmarkCategory(std::string const & nam
   return groupId;
 }
 
+void BookmarkManager::ReloadBookmarkCategory(kml::MarkGroupId & groupId, kml::CategoryData && data, bool autoSave /* = true */)
```

Comment:
Would it be better to use "UpdateBookmarkCategory", which will imply a following merging implementation that won't delete/recreate items if not needed, but only compare/merge/update them?

Reply:
UpdateBookmarkCategory may be the separate method when we implement the merging process.
But ReloadBookmarkCategory does only reloading.
Maybe both methods will be needed in some cases.

Reply:
Which cases do you see that may need reloading?

Reply:
Maybe in case of some failure or smth?
I can remane it and add the comment that the current implementation reload the collection.